doc/release-notes/9.0.md

```diff
@@ -0,0 +1,263 @@
+- [#1279](https://github.com/gluster/glusterfs/issues/1279) Fix several signed integer overflows
+- [#1278](https://github.com/gluster/glusterfs/issues/1278) Fix memory leak in afr_priv_destroy()
+- [#1275](https://github.com/gluster/glusterfs/issues/1275) Make glusterfs compile on all recent and supported versions o...
+- [#1272](https://github.com/gluster/glusterfs/issues/1272) tests/bugs/glusterd/mgmt-handshake-and-volume-sync-post-glust...
+- [#1269](https://github.com/gluster/glusterfs/issues/1269) common-ha: ganesha-ha.sh bad test for {rhel,centos} for pcs o...
+- [#1263](https://github.com/gluster/glusterfs/issues/1263) Fix memory leak in glusterd_store_retrieve_bricks()
+- [#1260](https://github.com/gluster/glusterfs/issues/1260) Implement minimal proper synchronization for gf_attach
+- [#1259](https://github.com/gluster/glusterfs/issues/1259) Fix memory leak in gf_cli_gsync_status_output()
+- [#1258](https://github.com/gluster/glusterfs/issues/1258) dht: Add null check
+- [#1255](https://github.com/gluster/glusterfs/issues/1255) Improve snapshot clone error message
+- [#1254](https://github.com/gluster/glusterfs/issues/1254) Prioritize ENOSPC over other lesser priority errors
+- [#1253](https://github.com/gluster/glusterfs/issues/1253) On Ovirt setup glusterfs performs poorly
+- [#1250](https://github.com/gluster/glusterfs/issues/1250) geo-rep: Fix corner case in rename on mkdir during hybrid crawl
+- [#1249](https://github.com/gluster/glusterfs/issues/1249) Drop 'const' type qualifier on return type
+- [#1248](https://github.com/gluster/glusterfs/issues/1248) Fix thread naming and related convention
+- [#1245](https://github.com/gluster/glusterfs/issues/1245) Spurious failures in ./tests/basic/ec/ec-quorum-count.t
+- [#1243](https://github.com/gluster/glusterfs/issues/1243) Modify and return iatt (especially size and block-count) in s...
+- [#1242](https://github.com/gluster/glusterfs/issues/1242) Rebalance - Improve Crawl time in rebalance
+- [#1240](https://github.com/gluster/glusterfs/issues/1240) tests/basic/afr/gfid-mismatch-resolution-with-fav-child-polic...
+- [#1236](https://github.com/gluster/glusterfs/issues/1236) glusterfs-geo-replication requires policycoreutils-python-uti...
+- [#1234](https://github.com/gluster/glusterfs/issues/1234) Fix ./tests/basic/fencing/afr-lock-heal-basic.t failure
+- [#1230](https://github.com/gluster/glusterfs/issues/1230) core dumped executing tests/line-coverage/errorgen-coverage.t
+- [#1228](https://github.com/gluster/glusterfs/issues/1228) seek functionalty is broken
+- [#1226](https://github.com/gluster/glusterfs/issues/1226) Gluster webhook update throws error
+- [#1225](https://github.com/gluster/glusterfs/issues/1225) fuse causes glusterd to dump core
+- [#1223](https://github.com/gluster/glusterfs/issues/1223) Failure of tests/basic/gfapi/gfapi-copy-file-range.t
+- [#1222](https://github.com/gluster/glusterfs/issues/1222) [RFE] Improve rebalance of sparse files
```

Comment:
this one could be mentioned as a nice feature.

Reply:
Done